odl/discr/discr_ops.py

```diff
@@ -338,12 +462,12 @@ class Gradient(Operator):
     """Spatial gradient operator for `DiscreteLp` spaces.
 
     Calls helper function `finite_diff` to calculate each component of the
-    resulting product space vector. For the adjoint of the
-    `Gradient` operator to match the negative `Divergence`
-    operator ``zero_padding`` is assumed.
+    resulting product space vector. For the adjoint of the `Gradient`
+    operator to match the negative `Divergence` operator zero padding is
+    assumed.
```

Comment:
For the adjoint of the `Gradient` operator, zero padding is assumed to match the negative `Divergence`.